Dockerfile

```diff
@@ -10,8 +10,9 @@ COPY dist ./dist
 # Copy server package.json for commands and dependencies
```

Comment:
Using node 11 up above in this Dockerfile is somewhat concerning. Probably out of scope for this migration, and may be lower hanging fruit for whenever a team can look at this next. Thanks for jumping on this app and getting it moved to K8S!

Reply:
Yup, we kept the same versions and only upgraded a few packages because of installation issues in the CI. We can look into upgrading node in the future and most of the packages are out of date (some >5 years).